.github/workflows/build_publish_lambda_layer.yml

```diff
@@ -65,6 +65,72 @@ jobs:
           name: ruby-layer-${{ matrix.arch }}.zip
           path: lambda/build/ruby-layer-${{ matrix.arch }}.zip
 
+  # extract the built layer from artifacts, then scan it with reverselab
+  reverselab_scan_layer:
+    needs:
+      - build_layer
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        arch:
+          - x86_64
+          - arm64
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - if: ${{ inputs.publish-dest == 'staging' }}
```

Comment:
https://github.com/solarwinds-cloud/solarwinds-apm-liboboe/blob/master/.github/workflows/publish.yml#L37-L41
You may want to check this above link to simplify the steps.

Reply:
Thanks, updated.